python/vyos/firewall.py

```diff
@@ -473,11 +474,20 @@ def parse_rule(rule_conf, hook, fw_name, rule_id, ip_name):
         if 'mark' in rule_conf['set']:
             mark = rule_conf['set']['mark']
             output.append(f'meta mark set {mark}')
+        if 'vrf' in rule_conf['set']:
+            set_table = True
+            vrf_name = rule_conf['set']['vrf']
+            # NOTE: VRF->table ID lookup depends on the VRF iface already existing. 
+            if vrf_name == 'default':
+                table = '254'
```

Comment:
'254' is a recurring number, please introduce a global variable referencing this number so we have a single point of contact. Other then that, LGTM


Can we import the magic value of 254 referenced in both firewall.py and policy_route.py from a single file? As it's policy based firewall.py should import the variable from policy_route.py

Reply:
Do you really want something in vyos.* to import from conf_scripts? 

There's an equivalent PR for policy local-route coming when I write the smoketests. Stashing the reference somewhere central does make sense. The standard tables & their IDs are used in a few spots by VRF configuration as well. 

Would vyos.defaults or vyos.utils.network be more appropriate? Something new and more specific?

Reply:
> Do you really want something in vyos.* to import from conf_scripts?

No, thanks for rethinking!

Please move it to `vyos.defaults`

Reply:
No its properly verbose - everyone knows what's what. Thanks

Reply:
Requested changes pushed.